### PR TITLE
fix resources

### PR DESCRIPTION
**Pull request by juliolustosa**

### CODE REVIEW

File: internal/provider/resource_tsuru_plan.go [hunk 1]
```diff
@@ -117,7 +118,6 @@ func resourceTsuruPlanRead(ctx context.Context, d *schema.ResourceData, meta int
 		return nil
 	}
 
-	d.SetId("")
```
Comment (wpjunior) [inline internal/provider/resource_tsuru_plan.go:120]: @juliolustosa could you revert this change ?

Comment (juliolustosa) [reply internal/provider/resource_tsuru_plan.go]: ok
